app/assets/stylesheets/govuk_publishing_components/components/_action-link.scss

```diff
@@ -125,10 +141,25 @@
   }
 }
 
+.gem-c-action-link--nhs {
+  &:before {
+    width: 80px;
+    height: 70px;
+    // sass-lint:disable no-duplicate-properties
+    background: image-url("govuk_publishing_components/action-link--nhs.png");
+    background: image-url("govuk_publishing_components/action-link--nhs.svg"), linear-gradient(transparent, transparent);
```

Comment:
is there a reason why we're adding the NHS image into the components gem? Can this not be passed in via the Application? I don't really see the benefit of putting this into the publishing components.

Reply:
It's a bit more complicated than just passing an image - the styling for this image is different from the rest of the component. We'd also need to apply styles from the application, which would break isolation.

If the image could be passed simply without additional styling then that would make sense.